src/halmos/__main__.py

```diff
@@ -819,19 +836,25 @@ def main() -> int:
     # compile
     #
 
-    try:
-        crytic_compile_args, crytic_compile_unknown_args  = crytic_compile_parser.parse_known_args()
+    build_cmd = [
+        'forge', # shutil.which('forge')
+        'build',
+        '--root', args.root,
+        '--extra-output', 'storageLayout', 'metadata'
+    ]
 
-        both_unknown = set(halmos_unknown_args) & set(crytic_compile_unknown_args)
-        if both_unknown:
-            print(color_warn(f'error: unrecognized arguments: {" ".join(both_unknown)}'))
-            return 1
+    # run forge without capturing stdout/stderr
+    build_exitcode = subprocess.run(build_cmd).returncode
```

Comment:
I think it's nice that we see the compiler output now 👌

```
[⠔] Compiling...
[⠒] Compiling 1 files with 0.5.17
[⠘] Compiling 43 files with 0.8.19
[⠢] Solc 0.5.17 finished in 9.82ms
[⠒] Solc 0.8.19 finished in 3.40s
```

Reply:
(and it makes it easier to troubleshoot if there's a compilation error too)

Reply:
yes, that was the intention.